EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationEvent.h

```diff
@@ -74,15 +74,20 @@ class VisualisationEvent
   }
   void remove_last_track() { mTracks.pop_back(); } // used to remove track assigned optimistically
 
-  // Adds visualisation cluser inside visualisation event
-  //void addCluster(const VisualisationCluster& cluster)
-  //{ mClusters.push_back(cluster); }
+  // Adds visualisation cluster inside visualisation event
   VisualisationCluster& addCluster(double XYZ[])
   {
     mClusters.emplace_back(XYZ);
```

Comment:
what is the reason to use doubles for VisualisationCluster? You just inflate the memory and jsons files (unless limiting precision of the output).

Reply:
json files store float. But you are right - I will change to float.